src/ui/view/dataverse/component/backButton/backButton.scss

```diff
@@ -23,11 +23,13 @@
       }
     }
 
-    .okp4-dataverse-portal-back-text {
-      @include noto-sans(700);
-      font-size: 14px;
-      line-height: 22px;
-      color: themed('text');
+    .okp4-dataverse-portal-button-main {
+      .okp4-dataverse-portal-button-label.outlined-tertiary {
```

Comment:
That is pure `CSS` syntax, not `SCSS`'s one 😉

Reply:
Ah yes, you are right, I completely missed this 😅